modules/end-user-guide/partials/proc_provisioning-jetbrains-activation-code-for-offline-use.adoc

```diff
@@ -1,55 +1,67 @@
 [id="provisioning-jetbrains-activation-code-for-offline-use_{context}"]
 = Provisioning JetBrains activation code for offline use
 
-An activation code for offline use is a file with a license code that can be retrieved from the license management section of your JetBrains account for the license that is assigned to you. When you purchase a personal subscription or are assigned a commercial subscription by your organization, you receive an email prompting you to create a JetBrains account that becomes connected with the license.
-
-NOTE: When using an activation code to activate a product, you need to generate a new activation code and apply it to your product each time the subscription is renewed.
+This section describes how to provision the JetBrains activation code to {prod-short}, when using link:https://www.jetbrains.com/store/[a JetBrains IDE requesting a registration]. After each subscription renewal, you need to generate an activation code and apply it to your product.
```

Comment:
```suggestion
Some editions of JetBrains IDEs require a paid subscription beyond the evaluation period, which means buying a license from JetBrains. To register a license, you need to provision to Che the JetBrains activation code for offline usage. When you renew your subscription, you will need to generate a new activation code and provision the new offline activation code.
```

Reply:
_To register a license, you need to provision_ ->
_To register a license, provision_

_When you renew your subscription, you will need to generate_ ->
_When renewing the subscription, generate_